src/System.Management.Automation/engine/CmdletParameterBinderController.cs

```diff
@@ -1064,134 +1058,56 @@ private bool RestoreParameter(CommandParameterInternal argumentToBind, MergedCom
         }
 
         /// <summary>
-        /// Binds the actual arguments to only the formal parameters
-        /// for only the parameters in the specified parameter set.
+        /// Validate the given named parameter against the specified parameter set,
+        /// and then bind the argument to the parameter.
         /// </summary>
-        /// <param name="parameterSets">
-        ///     The parameter set used to bind the arguments.
-        /// </param>
-        /// <param name="arguments">
-        ///     The arguments that should be attempted to bind to the parameters of the specified
-        ///     parameter binder.
-        /// </param>
-        /// <exception cref="ParameterBindingException">
-        /// if multiple parameters are found matching the name.
-        /// or
-        /// if no match could be found.
-        /// or
-        /// If argument transformation fails.
-        /// or
-        /// The argument could not be coerced to the appropriate type for the parameter.
-        /// or
-        /// The parameter argument transformation, prerequisite, or validation failed.
-        /// or
-        /// If the binding to the parameter fails.
-        /// </exception>
-        private Collection<CommandParameterInternal> BindParameters(uint parameterSets, Collection<CommandParameterInternal> arguments)
+        protected override void BindNamedParameter(
```

Comment:
(As side note, there is a full mess with 'bind the argument' vs 'bind the parameter' in comments and method names.)

Reply:
When talking about parameter binding, it should be `bind an argument to a parameter`, `an argument is bound (to a parameter)`, `a parameter is bound (by an argument)`.